### PR TITLE
create boxes controller

### PR DESCRIPTION
**Pull request by menakadevisundaramoorthy**
add new and create

### CODE REVIEW

**Review by Jordan-Patel — COMMENTED** [review]
@box.user = current_user will cause an error as the user is associated with the UserCollection, not the box so no need for this line.

File: app/controllers/boxes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def new
+    @box = Box.new
+  end
+
+  def create
+    @box = Box.new(params[box_params])
+    @box.user = current_user
+    @box.collection = Box.find(params[:collection_id])
+    if @box.save
+      redirect_to booking_path(@box)
```
Comment (Akki-on-Rails) [inline app/controllers/boxes_controller.rb:11]: here your path is wrong (booking)

**Review by Akki-on-Rails — CHANGES_REQUESTED** [review]
wrong path for create method

**Review by Akki-on-Rails — APPROVED** [review]

**Review by Jordan-Patel — APPROVED** [review]
Looks good.

**Pull request merged**